src/routes/user-vocabulary-list/UserVocabularyListScreen.tsx

```diff
@@ -78,8 +79,8 @@ const UserVocabularyListScreen = ({ navigation }: Props): ReactElement => {
         {!isKeyboardVisible && (
           <ButtonContainer>
             <Button
-              onPress={() => navigation.navigate('UserVocabularyOverview')}
-              label={getLabels().userVocabulary.create}
+              onPress={() => navigation.navigate('UserVocabularyEdit', { headerBackLabel: list })}
```

Comment:
hmh, `edit` is not really a good naming for "edit" or "create". I do not have a good idea though.

Reply:
what about `process` ? i really don't want to block this pr due to naming ;)

Reply:
Fine for me. Yes let's get this merged soon, can change naming afterwards, if needed.

Reply:
renamed.